application.go

```diff
@@ -304,10 +314,12 @@ func (client *Client) SetApplicationVersion(name string, version *ApplicationVer
 	client.log("SetApplicationVersion() setting the application: %s to version: %s", name, version)
 	uri := fmt.Sprintf("%s/%s", MARATHON_API_APPS, trimRootPath(name))
 	deploymentId := new(DeploymentID)
-	if err := client.apiPut(uri, version, deploymentId); err != nil {
+	err := client.apiPut(uri, version, deploymentId)
+	if err != nil {
```

Comment:
AFAIK the `if err := doSomething; err != nil { ..` is acceptable pattern in golang: http://blog.golang.org/error-handling-and-go
In this post all the single error assignments (`err :=`) are written in the `if err :=` manner.